common/src/main/scala/explore/components/Tile.scala

```diff
@@ -3,17 +3,30 @@
 
 package explore.components
 
+import cats.syntax.all._
 import explore.Icons
 import explore.components.ui.ExploreStyles
+import explore.model.enum.TileSizeState
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.common._
+import react.common.implicits._
 import react.semanticui.collections.menu._
+import react.semanticui.elements.button.Button
 
 final case class TileButton(body: VdomNode)
 
-final case class Tile(title: String, movable: Boolean, back: Option[TileButton])
-    extends ReactPropsWithChildren[Tile](Tile.component)
+final case class Tile(
+  title:             String,
+  back:              Option[TileButton] = None,
+  canMinimize:       Boolean = false,
+  canMaximize:       Boolean = false,
+  state:             TileSizeState = TileSizeState.Normal,
+  sizeStateCallback: TileSizeState => Callback = _ => Callback.empty
```

Comment:
These two parameters can be conmbined into a `View` (and probably use `ViewF.fromState` at the call site).

Reply:
That's a good idea, let me try that